synctl

```diff
@@ -2355,6 +2271,272 @@ class SyntheticTest(Base):
+        elif update_result.status_code == 400:
+            print(f'Error: {update_result}', update_result.content)
+        elif update_result.status_code == 429:
+            self.exit_synctl(-1, "Too Many Requests")
+        else:
+            print(
+                f'update test {test_id} failed, status code: {update_result.status_code}')
+
+    def update_a_smart_alert(self, alert_id, new_payload):
+        """API https://instana.github.io/openapi/#operation/updateSyntheticAlertConfig"""
+        self.check_host_and_token(self.auth["host"], self.auth["token"])
+        host = self.auth["host"]
+        token = self.auth["token"]
+        if new_payload is None:
+            print("config cannot be empty")
+            sys.exit(-1)
+
+        if alert_id is None or alert_id == "":
+            print("alert id should not be empty")
+            return
+        else:
+            update_url = f"{host}/api/events/settings/global-alert-configs/synthetics/{alert_id}"
+
+        headers = {
+            'Content-Type': 'application/json',
+            "Authorization": f"apiToken {token}"
+        }
+
+        update_result = requests.post(update_url,
+                                      headers=headers,
+                                      data=new_payload,
+                                      timeout=60,
+                                      verify=self.insecure)
+
+        if update_result.status_code == 200:
+            print(f"{alert_id} updated")
+        elif update_result.status_code == 400:
+            print(f'Error: {update_result}', update_result.content)
+        elif update_result.status_code == 429:
+            self.exit_synctl(-1, "Too Many Requests")
+        else:
+            print(
+                f'update alert {alert_id} failed, status code: {update_result.status_code}, {update_result.text}')
+
+    def update_using_file(self, file):
+        with open(file, 'rb') as json_file:
+            payload = json_file.read()
+            return payload
+
+    def update_label(self, payload, label):
+        """label"""
+        if label is None:
+            print("no label")
+        else:
+            payload["label"] = label
+            return payload
+    def update_description(self, payload, description):
+        """description"""
+        if description is None:
+            print("no description")
+        else:
+            payload["description"] = description
+            return payload
+
+    def update_record_video(self, payload, record_video=False):
+        if record_video is not None and record_video.upper() in ("TRUE", "FALSE"):
+            payload["configuration"]["recordVideo"] = record_video
+            return payload
+
+    def update_active(self, payload, active):
+        """active"""
+        if active is None:
+            return
+        if active == "false":
+            payload["active"] = False
+        if active == "true":
+            payload["active"] = True
+            return payload
```

Comment:
There is an issue for this function, if active == false, no value is returned. So return payload outside if clause.
```suggestion
        return payload
```